### ISSUE TITLE
False positive error on sass variables and on custom properties

### ISSUE THREAD

**Issue by Semigradsky** [ismay/stylelint-no-unsupported-browser-features#136]
.browserslistrc:
```
> 1%
not IE 11
```
Code example:
```scss
:root {
  $column-count: 3; // error

  --column-count: 3; // error
}
```
Errors:
> Unexpected browser feature "multicolumn" is only partially supported by Edge 86, Firefox 81, Chrome 85,86, Chrome for Android 86 (plugin/no-unsupported-browser-features)

**Comment by Semigradsky** [comment]
Looks like it is [one of common edge-case](https://stylelint.io/developer-guide/rules#commonly-overlooked-edge-cases) =)

**Comment by Semigradsky** [comment]
> Doiuse and this plugin are only compatible with standard css syntax, so syntaxes like scss, less and others aren't supported.

Sorry, I didn't pay attention to this notice. In any case 'custom properties' is standard css syntax.

**Comment by ismay** [comment]
Hi, thanks for the issue. Yes, this plugin only supports vanilla css, so it's possible it behaves incorrectly with sass.

And you're right, custom properties are standard syntax. Doiuse, the library that this plugin uses for detecting css support, is the lib responsible for the actual detecting of support. This plugin only wraps it.

The false positive for css variables is a known issue: https://github.com/anandthakker/doiuse/issues/82. If you want to fix this or discuss it then the doiuse repo is the place to do it :). Hope that helps! I'll close this since there isn't anything I can do about it from this lib.

**Comment by ismay** [comment]
See also this comment, which explains the above in a bit more detail (though it's also partially about another topic): https://github.com/ismay/stylelint-no-unsupported-browser-features/issues/45#issuecomment-354135445
